torch/_dynamo/backends/cudagraphs.py

```diff
@@ -143,7 +143,7 @@ def forward_cudagraphs(aot_model, aot_inputs, is_inference=False):
             range(fixed),
             device_index=boxed_device_index.value,
             is_backward=False,
-            is_inference=False,
+            is_inference=is_inference,
```

Comment:
@eellison passing is_inference wasn't working correctly for the cudagraphs backend. This meant that users had to manually use https://pytorch.org/docs/stable/generated/torch.compiler.cudagraph_mark_step_begin.html in order to get good inference performance previously. Let's consider this some evidence that the cudagraphs backend isn't too popular and inductor is much more valuable to work on.